src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.RoslynWorkspace/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
I don't know why, but this analyzer wasn't working on my machine. Will log a follow up to fix this later, unless the build fails :)

Reply:
Well that's unfortunate ... 

I'm sitting here trying to think about how we fix this. Think I'm just gonna try and build locally and see what the damage is

Reply:
I've turned that option to warn before. It's awful. The PublicAPI files are just horribly incomplete.

Reply:
Uggg, when we delete this public API checking is turned on for all our test projects.